sections/leaderboard/Leaderboard/Leaderboard.tsx

```diff
@@ -166,13 +170,35 @@ const Leaderboard: FC<LeaderboardProps> = ({ compact }: LeaderboardProps) => {
 											<></>
 										),
 										accessor: 'trader',
-										Cell: (cellProps: CellProps<any>) => (
-											<StyledOrderType onClick={() => onClickTrader(cellProps.row.original.trader)}>
-												{compact && cellProps.row.original.rank + '. '}
-												<StyledTrader>{cellProps.row.original.traderShort}</StyledTrader>
-												{getMedal(cellProps.row.index + 1)}
-											</StyledOrderType>
-										),
+										Cell: (cellProps: CellProps<any>) => {
+											const { ensName, ensAvatar } = useENS(cellProps.row.original.trader);
+											return (
+												<StyledOrderType
+													onClick={() => onClickTrader(cellProps.row.original.trader, ensName)}
+												>
+													{compact && cellProps.row.original.rank + '. '}
+													<StyledTrader>
+														{ensName ? (
+															<>
+																{ensAvatar && (
+																	<img
+																		src={ensAvatar}
+																		alt={ensName}
+																		width={16}
+																		height={16}
```

Comment:
The design token is a multiple of `8`.

and the size of the OP icon in the NetworkSwitcher modal is `17` px.

So I set the ENS avatar size to `16` px (8 x 2).